src/test/crimson/seastar_runner.h

```diff
@@ -59,10 +72,19 @@ struct SeastarRunner {
 	on_end.reset();
       });
     });
+    if (ret != 0) {
+      std::cerr << "Seastar app returns " << ret << std::endl;
+    }
+    if (!begin_signaled) {
+      ::eventfd_write(begin_fd.get(), 1);
```

Comment:
how about writing a different value to signal that the app is not started at all?

Reply:
done.